pkg/util/github/pullrequest_test.go

```diff
@@ -0,0 +1,126 @@
+package github_test
```

Comment:
may be the pkg name is `github`

Reply:
I think it's OK. 
`Test files that declare a package with the suffix "_test" will be compiled as a
separate package, and then linked and run with the main test binary.`

Refer to https://pkg.go.dev/cmd/go/internal/test

Reply:
First time I've seen it used in this way. I've looked at several go open source projects and haven't seen this way.
https://github.com/moby/moby/tree/master/pkg/archive
https://github.com/kubernetes/kubernetes/tree/master/pkg/kubelet

I think the `_test` suffix describes the file name but not the package name.

Reply:
> may be the pkg name is `github`

I agree with you. @steinliber and I usually remove "_test" suffix, or you have to exported every function and vars you want to test.

Reply:
Go have 2 types test:
1 internal test(package name don't have suffix `_test`): mainly use to test unexported function.
2 external test(package name have suffix `_test`): mainly use to test exported function.

In this case, I prefer to use the external test for this package, and `ginkgo` also generates a test file with a package name suffix '_test'.